### PR TITLE
Development page added

### PR DESCRIPTION
**Pull request by berakrishnendu36**

### CODE REVIEW

**Review by raydwaipayan — CHANGES_REQUESTED** [review]
Please put package-lock.json into .gitignore.

Rest some changes and we are good to go.

File: pages/chapters/development.vue [hunk 1]
```diff
@@ -0,0 +1,261 @@
+<template>
+  <div>
+    <div
+      class="hero is-centered has-text-centered is-vcentered has-background-black"
+    >
+      <div id="app">
+        <vue-particles
```
[the remark this answers is not in the document]
Comment (berakrishnendu36) [reply pages/chapters/development.vue:7]: okayy.. 😅

**Comment by raydwaipayan** [comment]
@raydwaipayan resolve merge conflict

**Comment by raydwaipayan** [comment]
Added via commit f4156f5f49c76b1d599c975fc725efc78ffaad82

**Pull request closed without merging**